test/scenarios-periodics/el92-yminus2@el94-crel@upgrade-ok.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Sourced from scenario.sh and uses functions defined there.
+
+scenario_create_vms() {
+    # Determine the starting image based on the source tree current
+    # version, minus one.
+    local start_image
+    start_image="rhel-9.2-microshift-4.$(yminus2_minor_version)"
+
+    prepare_kickstart host1 kickstart.ks.template "${start_image}"
+    launch_vm host1
```

Comment:
```suggestion
	launch_vm host1 rhel-9.2
```
+ Change `DEFAULT_BOOT_BLUEPRINT` to use `rhel-9.4` in scenario.sh